### ISSUE TITLE
Unable to import library as a maven dependency (according to readme code snippet).

### ISSUE THREAD

**Issue by roosemberth** [pryv/lib-java#31]
I'm using the code snippet on the Readme:
```
<dependency>
  <groupId>com.pryv</groupId>
  <artifactId>java</artifactId>
  <version>1.0.2</version>
  <type>pom</type>
</dependency>
```
But I receive the error
    Failure to find com.pryv:java:pom:1.0.2 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced

When searching on the maven repository, I realize the last updated version is 0.1.0, which is not in in this git repository. Can you please update it or modify the readme to get an up-to-date version of the dependency from elsewhere?

cf. http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.pryv%22

Regards,
Roos

**Comment by tmodoux** [comment]
Hi Roos,

Thanks for your notification.
Effectively, versions 1.x.x are not on maven central yet, it requires an additional validation process, I am currently working on this.

In the meantime, you can use Gradle or look at https://bintray.com/techpryv/maven/pryv-lib/1.0.2, it contains the sources (.jar, .aar, .pom) under Files tab, maybe you can do something with this as a quick fix.
Another option would be to use our git repo as dependency, I know it is possible in Gradle but I am not sure about Maven.

Anyway, I will keep you posted here about the progress of the Maven central publication.

**Comment by tmodoux** [comment]
Hi back,

I just published a version 1.0.3, which is now available on Maven central (see the updated [Readme](https://github.com/pryv/lib-java)).
Please tell me if everything is ok and I will close the issue :)

Thiébaud

**Comment by roosemberth** [comment]
Hello,
I just made a clean build. It's working, thanks!


Roos
